tools/generator/SourceWriters/BoundFieldAsProperty.cs

```diff
@@ -161,7 +163,6 @@ protected override void WriteSetterBody (CodeWriter writer)
 			} else {
 				writer.WriteLine ($"{(invokeType != "Object" ? arg : "new JniObjectReference (" + arg + ")")});");
 			}
-			writer.WriteLine ();
```

Comment:
Not sure why I added this in the first place, and it's presence causes extra whitespace changes when comparing src/Mono.Android build output from pre- a65d6fb4d79041c959435c15dc3dcf5331e5d715 vs. post- a65d6fb4d79041c959435c15dc3dcf5331e5d715…